bindings/python/spatial/explog.hpp

```diff
@@ -53,6 +53,11 @@ namespace pinocchio
       return res;
     }
     
+    Eigen::Vector3d log3_proxy(Eigen::Matrix3d R)
```

Comment:
Use a reference here then.

Reply:
Oh! Don't know how I did that! Will fix it immediately